14-include.md

```diff
@@ -1,29 +1,27 @@
-## Dependency on other compose projects
+## Include
 
-A Compose application can declare dependency on another compose application, managed by another team
-or shared with others. This allows to keep a compose file reasonably complicated for the limited
-amount of resources a single team has to declare for it's own sub-domain within a larger deployment.
+A Compose application can declare dependency on another Compose application that is managed by another team
+or shared with others. This is useful if teams need to keep a Compose file reasonably complicated for the limited
+amount of resources it has to declare for it's own sub-domain, within a larger deployment.
 
-`include` top-level section is used to define dependency to another compose application (or subdomain).
-Each path listed in this section must be loaded as an individual compose model with it's own project
-directory to resolve relative paths. 
+The `include` top-level section is used to define the dependency on another Compose application, or sub-domain.
+Each path listed in the `include` section is loaded as an individual Compose application model, with it's own project directory, in order to resolve relative paths. 
 
-Once the application to be included has been loaded, all resources definitions are copied into the 
-current compose application model. Compose must warn user if some resource name conflict, and not 
-try to merge those. To enforce this, `include` must be evaluated after the compose file(s) selected 
-by user to define the Compose application model have been parsed and merged, so that conflicts 
-between included compose files and those selected are detected.
+Once the included Compose application is loaded, all resources definitions are copied into the 
+current Compose application model. Compose displays a warning if resource names conflict and doesn't 
+try to merge them. To enforce this, `include` is evaluated after the Compose file(s) selected 
+to define the Compose application model have been parsed and merged, so that conflicts 
+between Compose files are detected.
 
-`include` applies recursively: an included compose file which declares it's own `include` section
-will trigger those other files to be included as well.
+`include` applies recursively so an included Compose file which declares its own `include` section, triggers those other files to be included as well.
 
 The resulting resources can be used in the including compose model for cross-services references.
 
-### short syntax
+### Short syntax
 
-Short syntax only defines path to another compose file. File is loaded with parent
-folder as project directory, and optional `.env` file being loaded to define variables default values
-for interpolation, while local project environment can override those values. 
+The short syntax only defines paths to another Compose file. The file is loaded with the parent
```

Comment:
```suggestion
The short syntax only defines paths to other Compose files. Each file is loaded with the parent
```
It currently reads like multiple paths to a single file because plural `paths` but singular `Compose file`. Not sure if the suggested change is correct.

Reply:
ah yes! good catch